webapp/src/app/core/modules/openapi/model/pull-request-review.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Hephaestus API
+ * API documentation for the Hephaestus application server.
+ *
+ * The version of the OpenAPI document: 0.0.1
+ * Contact: felixtj.dietrich@tum.de
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+import { User } from './user';
+import { PullRequest } from './pull-request';
+
+
+export interface PullRequestReview { 
+    id?: number;
+    createdAt?: string;
+    updatedAt?: string;
+    author?: User;
+    state: PullRequestReview.StateEnum;
+    submittedAt?: string;
+    pullRequest?: PullRequest;
+}
+export namespace PullRequestReview {
+    export type StateEnum = 'COMMENTED' | 'APPROVED' | 'CHANGES_REQUESTED' | 'DISMISSED';
```

Comment:
OPEN?

Reply:
these are the auto-generated files by OpenAPI, which use our Java-files as input. So this file is the Typescript equivalent of the `PullRequestReviewState.java`.